### ISSUE TITLE
Maker Select 3D/Cura 2.3/2.4.  "The printer is not connected"

### ISSUE THREAD

**Issue by TTVert** [daid/LegacyCura#1556]
[Logs.zip](https://github.com/daid/LegacyCura/files/788076/Logs.zip)
win10 64Bit, selected a Prusa i3 but also tried to create a custom printer to no avail, Works fine on Cura 15.0.4.6 and I confirmed I can communicate w/ COM3 at 115200.  It seems others have this same issue and I'm not sure where to look. Looking at the log files in  USBPrinting don't even mention it looking to any port.  It appears this is all auto configured now but I cannot find a log indicating what it is trying to do or where it is failing.  

I've attached the two log files I'm aware of that may have something to do with this but any other help would be appreciated.

Thanks
Dave

**Comment by Ghostkeeper** [comment]
Your attempt to attach log files failed. At least, I don't see any.

What version of Cura are you using? We haven't released any `Cura 3.21`.

**Comment by BagelOrb** [comment]
We at Ultimaker don't really support 3rd party printers.
We accept pull requests though ;)

**Comment by TTVert** [comment]
Fair enough, but why list printers as options to choose that clearly don't work?

Dave

**Comment by TTVert** [comment]
Sorry Ghostkeeper, I had to zip them as log files weren't allowed. I've attached.

**Comment by Ghostkeeper** [comment]
We don't have those 3rd party printers, so we can't test whether they still work other than trying to slice with them just in the interface. Stuff like the printer not connecting is impossible to test without buying the actual printer.

In short, we don't know that they don't work. We rely on external contributors such as you to report problems with them and to fix the connection.

**Comment by TTVert** [comment]
If I could fix them I'd design the software myself.  In this case the software is auto detecting hardware however it isn't even trying to auto detect usb devices per the log.  All I need is the ability to manually specify the COM port and baud rate w/in your software.  Can this somehow be done w/in your software?  Is there a place within the config files of your software to add a line of code to manually specify a COM port and bypass the auto detection which clearly isn't even attempting to auto detect usb devices?  Your 15.x software allows this and the printer prints fine.

**Comment by BagelOrb** [comment]
This issue is logged under the repo of Cura 1.15.04.

You should log your issue under github.com/Ultimaker/Cura so that it actually arrives at the Cura team which is currently developing Cura 2.

**Comment by Ghostkeeper** [comment]
It was also logged under the Cura repository, BagelOrb: https://github.com/Ultimaker/Cura/issues/1434

**Comment by TTVert** [comment]
Yeah sorry I had it in the wrong place, fairly new to github.  So far no luck figuring anything out.
